libjcat/jcat-ed25519-engine.c

```diff
@@ -132,10 +204,12 @@ jcat_ed25519_engine_pubkey_sign(JcatEngine *engine,
 				JcatSignFlags flags,
 				GError **error)
 {
-	Ed25519Key pubkey = {0};
-	Ed25519Sig privkey = {0};
-	Ed25519Sig sig = {0};
 	g_autoptr(GBytes) blob_sig = NULL;
+	g_auto(gnutls_pubkey_t) pubkey = NULL;
+	g_auto(gnutls_privkey_t) privkey = NULL;
+	gnutls_datum_t data;
```

Comment:
can we `= {0}` here just to be on the safe side?

Reply:
Fixed.